### PR TITLE
Update `insert-ignores.sh` to current requirements #1450

### PR DESCRIPTION
**Pull request by hgvanpariya**
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: scripts/insert-ignores.sh [hunk 1]
```diff
@@ -4,12 +4,22 @@ for file in `find . -name "*Test.java"`; do
...
   echo -e "\n\n\n*** $file ******************************"
-  cat $file | \
-  sed 's/import org.junit.Test;/import org.junit.Test;\
-import org.junit.Ignore;/' | \
-  sed 's/@Test/@Ignore\
-    @Test/' | \
-  sed '1,/@Ignore/s/.*@Ignore//' > "$tempfile" \
-  && mv "$tempfile" "$file"
+
+#ignoreFound variable will check if already Ignore is imporeted or not.
```
Comment (sjwarner-bp) [inline scripts/insert-ignores.sh]: Typo -> `imporeted` should be `imported`

File: scripts/insert-ignores.sh [hunk 2]
```diff
@@ -4,12 +4,22 @@ for file in `find . -name "*Test.java"`; do
...
+ignoreFound=`grep -q 'import org.junit.Ignore;' $file ; echo $?` > /dev/null
+
+#if ignore is not imported then add line for import
+if [ $ignoreFound -ne 0 ]
+then
+  cat $file | sed 's/import org.junit.Test;/import org.junit.Test;\
+import org.junit.Ignore;/' >  $tempfile
+  mv "$tempfile" "$file"
+pwd
```
Comment (sjwarner-bp) [inline scripts/insert-ignores.sh]: Should this be indented?

**Comment by hgvanpariya** [comment]
@sjwarner-bp : thank you for your comments ... I'll update it ...

**Comment by hgvanpariya** [comment]
@sjwarner-bp : updated the script... please validate once...

**Comment by sjwarner-bp** [comment]
Also to add to the comment above, it seems like the indentation might be slightly wrong :slightly_smiling_face:

**Comment by hgvanpariya** [comment]
@sjwarner-bp , I've updated the script with following two changes : 
1. It will not add import `import org.junit.Ignore;` if any type of `@Ignore` is not found in the file
2. indentation is fixed. ( line no. 31, is out of indentation, because, it will add extra space in the Test file it I'll add Tab in the beginning )

Please let me know if any further changes are needed.

**Comment by sjwarner-bp** [comment]
Thanks for all the improvements with this! I'll pull it down and try it out at lunchtime :slightly_smiling_face:

**Comment by sjwarner-bp** [comment]
Thanks @hgvanpariya :fireworks:

**Pull request merged**